skl2onnx/operator_converters/one_vs_one_classifier.py

```diff
@@ -0,0 +1,175 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from sklearn.base import is_regressor
+from ..proto import onnx_proto
+from ..common._apply_operation import (
+    apply_concat, apply_identity, apply_mul, apply_reshape, apply_transpose)
+from ..common._registration import register_converter
+from ..common._topology import Scope, Operator
+from ..common._container import ModelComponentContainer
+from ..common._apply_operation import apply_normalization
+from ..common._apply_operation import (
+    apply_slice, apply_sub, apply_cast, apply_abs, apply_add, apply_div)
+from ..common.utils_classifier import _finalize_converter_classes
+from ..common.data_types import guess_proto_type, Int64TensorType, FloatTensorType
+from .._supported_operators import sklearn_operator_name_map
+
+
+def convert_one_vs_one_classifier(scope: Scope, operator: Operator,
+                                   container: ModelComponentContainer):
+
+    proto_dtype = guess_proto_type(operator.inputs[0].type)
+    if proto_dtype != onnx_proto.TensorProto.DOUBLE:
+        proto_dtype = onnx_proto.TensorProto.FLOAT
+    op = operator.raw_operator
+    options = container.get_options(op, dict(raw_scores=False))
+    use_raw_scores = options['raw_scores']
+    label_names = []
+    score_names = []
+    for i, estimator in enumerate(op.estimators_):
+        op_type = sklearn_operator_name_map[type(estimator)]
+        label_name_out = 'label_out_%d' % i
+        score_name_out = 'score_out_%d' % i
+        if op_type == 'SklearnLinearSVC':
```

Comment:
You should something like that: https://github.com/onnx/sklearn-onnx/blob/main/skl2onnx/operator_converters/one_vs_rest_classifier.py#L37.